src/components/FormMessage/FormMessage.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * @file FormMessage.js
+ * Exports a component that renders a given form message.
+ */
+
+import React, { Fragment } from 'react';
+import PropTypes from 'prop-types';
+import { Typography } from '@material-ui/core';
+
+const FormMessage = ({ children }) => (
+  <Fragment>
```

Comment:
Is this needed considering ultimately, you return one element?

Reply:
Good point, I adjusted it: